components/apimgt/org.wso2.carbon.apimgt.rest.api.admin.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/admin/v1/impl/OrgThemesApiServiceImpl.java

```diff
@@ -0,0 +1,108 @@
+package org.wso2.carbon.apimgt.rest.api.admin.v1.impl;
+
+import org.apache.commons.io.FileUtils;
+import org.wso2.carbon.apimgt.api.APIManagementException;
+import org.wso2.carbon.apimgt.api.ExceptionCodes;
+import org.wso2.carbon.apimgt.impl.APIAdminImpl;
+import org.wso2.carbon.apimgt.impl.APIConstants;
+import org.wso2.carbon.apimgt.rest.api.admin.v1.*;
```

Comment:
You cant have wild card imports. Import the relevant class.

Reply:
Addressed by [Add 3rd set of requested changes](https://github.com/wso2/carbon-apimgt/pull/12874/commits/7c12fe98746e3df262f2a9d66cef004344c718c7)